ports/ofmiceandmoggies/Of Mice and Moggies.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+get_controls
+
+GAMEDIR=/$directory/ports/ofmiceandmoggies
+CONFDIR="$GAMEDIR/conf/"
+
+# Ensure the conf directory exists
+mkdir -p "$GAMEDIR/conf"
+
+# Set the XDG environment variables for config & savefiles
+export XDG_DATA_HOME="$CONFDIR"
+export SDL_GAMECONTROLLERCONFIG="$sdl_controllerconfig"
+
+cd $GAMEDIR
+
+# Patch file
+if [ -f "./gamedata/Of Mice and Moggies.pck" ]; then
+  pm_message "Patching Of Mice and Moggies.pck"
+  $controlfolder/xdelta3 -d -s "./gamedata/Of Mice and Moggies.pck" "./gamedata/patch.xdelta3" "./gamedata/Of Mice and Moggies-patched.pck"
+  [ $? -eq 0 ] && rm "./gamedata/Of Mice and Moggies.pck" || pm_message "Patching of Of Mice and Moggies.pck has failed"
+  # Delete unneeded files
+  rm -f gamedata/*.exe
+fi
+
+# Load runtime
+runtime="frt_3.3.4"
+if [ ! -f "$controlfolder/libs/${runtime}.squashfs" ]; then
+  # Check for runtime if not downloaded via PM
+  if [ ! -f "$controlfolder/harbourmaster" ]; then
+    pm_message "This port requires the latest PortMaster to run, please go to https://portmaster.games/ for more info." > /dev/tty0
```

Comment:
Does pm_message need the directed terminal console output ( > /dev/tty0)?

Reply:
Note sure. @Cebion ?